### DISCUSSION TITLE
django-rest-authemail

### DISCUSSION THREAD

**Discussion by anatoliypay** [community/community#60894] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!. First. Sorry for my english:)) I have used this library.When the verification email arrives, the link looks like this "http://127.0.0.1:8000/signup/verify/......" , but the verification url is "http://127.0.0.1:8000/api/accounts/signup/verify/.....". How to fix it? Sometimes irun an application on the local network at 192.168.219.107 and the link still says 127.0.0.1. Answer, please. I am a beginer developer.

**Comment by AbdrrahimDahmani** [comment]
you issue seems to be related to the configuration of your application's email settings. Try to find the code that sends the verification email, it may be located on the code part of responsible of authentication, then look for the code that generates the uri and try to replace "http://127.0.0.1:8000/signup/verify/" with "http://127.0.0.1:8000/api/accounts/signup/verify/".
If you can provide you project repository I might assist you!

**Comment by anatoliypay** [comment]
This is my test project "https://github.com/anatoliypay/django-email.git". Please help me. because its the only library i lerned out on my own. im beginer.

**Comment by AbdrrahimDahmani** [comment]
@anatoliypay Ive seen your code and I think that you should modify the urlPatterns in the config/urls.py file and things should work
![image](https://github.com/community/community/assets/80603216/31c43bdf-9dcf-4dfd-8753-b0ccf5740f07)

**Comment by Pasindusameera** [comment]
Hello! Don’t worry about your English 😊
This problem happens because your backend is generating the email link using a default host (127.0.0.1), not the actual address where your app is running.

Why this happens

127.0.0.1 means localhost

Your email verification library builds the URL automatically

It does not know whether you are running on
127.0.0.1, 192.168.x.x, or a real domain

So it always falls back to 127.0.0.1
